test/cloudformation/timestream_cloudformation.yaml

```
@@ -26,11 +29,21 @@ Resources:
       RetentionProperties:
         MemoryStoreRetentionPeriodInHours: "2160" # Three months of memory store retention
         MagneticStoreRetentionPeriodInDays: "1825" # 5 years of magnetic store retention
+  ResourceTimestreamTable:
```

```suggestion
  SweeperTimestreamTable:
```

Just because it refers to the sweeper cleanup